bodhi/server/tasks/updates.py

```diff
@@ -95,6 +97,19 @@ def run(self, api_version: int, data: dict):
 
             bugs = []
             if action == "edit":
+                # When editing a Pending update that has all its build signed and not composed
+                # by bodhi (rawhide) move it to directly to Testing.
+                if not update.release.composed_by_bodhi \
+                        and update.status is UpdateStatus.pending \
```

Comment:
Why do you use `is` here instead of `==`?

Reply:
I think in this case it doesn't matter.

https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is

Reply:
For no good reason :-)